foundation-models/openai/src/test/resources/chatCompletionResponse.json

```diff
@@ -24,13 +25,13 @@
           "filtered": false,
           "severity": "safe"
         },
-        "selfHarm": null,
+        "self_harm": {
```

Comment:
looks like [spec](https://github.com/Azure/azure-rest-api-specs/blob/30ec6607bbcbeceb13b634de955667c5aa45766a/specification/cognitiveservices/data-plane/AzureOpenAI/inference/stable/2024-02-01/inference.json#L1022) was corrected to proper snake_case